include/dem/from_file_insertion.h

```diff
@@ -0,0 +1,108 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2019 by the Lethe authors
```

Comment:
2019 - 2024, but Bruno says we will have a script for that, but since it's a new file I think it's better to have the current date ?

Reply:
eventualy we will overwrite all of them, I just have not had the time yet to do it. We'd just need a week with no open PRs